azure-devops/azext_devops/dev/repos/pull_request.py

```diff
@@ -436,6 +437,29 @@ def add_pull_request_work_items(id, work_items, organization=None, detect=None):
     return wit_client.get_work_items(ids=ids)
 
 
+def checkout(id):  # pylint: disable=redefined-builtin
+    """Checkout the PR source branch locally, if no local changes are present
+    :param id: ID of the pull request.
+    :type id: int
+    """
+    organization = resolve_instance(detect='on', organization=None)
```

Comment:
resolve_instance will throw first. User will see "--organization must be specified..." instead of your error if they try from outside a git repo context.

Reply:
nice catch